networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/jvmlibp2p/gossip/AttestationTopicHandler.java

```diff
@@ -33,25 +34,32 @@
 public class AttestationTopicHandler extends GossipTopicHandler<Attestation> {
 
   private static final Logger LOG = LogManager.getLogger();
-  private static final Topic ATTESTATIONS_TOPIC = new Topic("/eth2/beacon_attestation/ssz");
+  private final Topic attestationsTopic;
   private final ChainStorageClient chainStorageClient;
 
+  private final int committeeIndex;
+
   protected AttestationTopicHandler(
```

Comment:
There are some additional validations to implement for the subnet attestations: https://github.com/ethereum/eth2.0-specs/blob/v0.9.1/specs/networking/p2p-interface.md#attestation-subnets

But I think we can handle that in a separate PR.

Reply:
Good catch, I'll actually handle that now.

Reply:
actually no, the validations for both attestation subnets and aggregation subnets look huge. so it's better to that in another PR, as you said.

Reply:
Made this issue for this: https://pegasys1.atlassian.net/browse/BC-119. I think this ties in well with the null checking for beacon state issue described above